goldenmaster/Plugins/TbEnum/Source/TbEnum/Private/TbEnumSettings.cpp

```diff
@@ -15,9 +15,39 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 #include "TbEnumSettings.h"
+#include "ApiGearSettings.h"
 
 UTbEnumSettings::UTbEnumSettings(const FObjectInitializer& ObjectInitializer)
 	: Super(ObjectInitializer)
-	, ServiceConnection(ETbEnumConnection(ETbEnumConnection::CONNECTION_LOCAL))
 {
 }
+
+void UTbEnumSettings::PostInitProperties()
+{
+	Super::PostInitProperties();
+
+	// if set to Local backend, we do not check any further
+	if (ConnectionIdentifier == "Local")
+	{
+		return;
+	}
+
+	// all other identifiers need to be validated against existing connection configurations
+	UApiGearSettings* settings = GetMutableDefault<UApiGearSettings>();
+
+	bool bConnectionIdentifierFound = false;
+	for (auto ConnectionSetting : settings->Connections)
```

Comment:
I'd maybe extract it to a function checkIfConnectionExists/Configured(connectionIdentifier), then you don't need that comment. or maybe rename the bool from line 38 isExpectedConnectionConfigured

Reply:
Created new issue to clean this up https://github.com/apigear-io/template-unreal/issues/53